stubs/tensorflow/tensorflow/autograph/__init__.pyi

```diff
@@ -0,0 +1,27 @@
+from collections.abc import Callable
+from typing import Any, TypeVar, overload
+from typing_extensions import ParamSpec
+
+from tensorflow.autograph.experimental import Feature
+
+_Param = ParamSpec("_Param")
+_RetType = TypeVar("_RetType")
+_Type = TypeVar("_Type")
+
+def set_verbosity(level: int, alsologtostdout: bool = False) -> None: ...
+def to_code(
+    entity: Callable[_Param, Any] | type,
+    recursive: bool = True,
+    experimental_optional_features: None | Feature | tuple[Feature, ...] = None,
+) -> str: ...
+@overload
```

Comment:
I don't see what these overloads add, as the second overload accepts everything the first one does.

Reply:
Fixed in baf73245d. The first overload was useless indeed...